### PR TITLE
remove emoji in favor of text

### PR DESCRIPTION
**Pull request by rv-egranger**
https://github.com/segmentio/evergreen/issues/582

https://github.com/segmentio/ui-box/issues/23

The box has felt very noisy in the DOM inspector and elsewhere.

### CODE REVIEW

**Comment by artmsilva** [comment]
⚡️ 👍 LGTM

File: src/enhancers/layout.ts [hunk 1]
```diff
@@ -51,9 +51,9 @@ export const propEnhancers: PropEnhancers = {
   boxSizing: (value: PropEnhancerValueType) => getCss(boxSizing, value),
   clear: (value: PropEnhancerValueType) => getCss(clear, value),
   clearfix: () => ({
-    className: '📦clearfix',
+    className: 'ui_box_clearfix',
```
Comment (mshwery) [inline src/enhancers/layout.ts]: in favor of brevity here I would suggest `box-*`, it's double the chars that the emoji is (`📦`= 2 vs `box-` = 4)

**Comment by mshwery** [comment]
We may want to consider this as a "breaking" change for anyone who may be relying on the old classnames for selection.

cc @Rowno @solon-aguiar @vadimdemedes

File: src/enhancers/layout.ts (hunk 1 above)
Comment (rv-egranger) [reply src/enhancers/layout.ts]: I could also do a ❒ as someone suggested in the issues. It won't take on native styling so it'd stick out much less in the DOM.

Comment (vadimdemedes) [reply src/enhancers/layout.ts]: You could also use `ub-*` (`ui-box-*`), as `box-*` sounds common enough to run into conflicts.

**Comment by vadimdemedes** [comment]
I'd consider this a breaking change for sure, since we don't know if people rely on existing selectors or not. I'm certainly supporting this change though! Emojis are annoying.

File: src/enhancers/layout.ts (hunk 1 above)
Comment (rv-egranger) [reply src/enhancers/layout.ts]: added `ub-`

**Comment by rv-egranger** [comment]
@mshwery do you need anything else from me? I didn't see a contributing guide so unsure on versioning etc.

**Comment by mshwery** [comment]
@rv-egranger i'll merge and we'll release as a new major version.

**Pull request merged**